src/components/ListTemplate/ColumnTemplates/DefaultColumn/DefaultColumn.js

```diff
@@ -0,0 +1,53 @@
+import React, { memo } from 'react';
+import HighLight from 'react-highlighter';
+import PropTypes from 'prop-types';
+
+import { AppIcon } from '@folio/stripes/core';
+
+import sharedCss from '../../../../shared.css';
+
+export const DefaultColumn = memo(props => {
+  const {
+    value,
+    className,
+    iconKey,
+    searchTerm,
+  } = props;
+
+  const recordContent = () => {
+    if (searchTerm) {
+      return value;
+    }
+
+    return (
+      <HighLight
+        search={searchTerm}
+        className={sharedCss.container}
+      >
+        {value}
+      </HighLight>
+    );
+  };
+
+  if (!iconKey) {
+    return recordContent();
+  }
+
+  return (
+    <AppIcon
+      size="small"
+      app="data-import"
+      iconKey={iconKey}
+      className={className || sharedCss.cellAppIcon}
+    >
+      {recordContent()}
+    </AppIcon>
+  );
+});
+
+DefaultColumn.propTypes = {
+  value: PropTypes.string.isRequired,
+  iconKey: PropTypes.string,
+  className: PropTypes.string,
+  searchTerm: PropTypes.string,
```

Comment:
Should it be required? I see that for MatchColumn it is. Please make it consistent and add default propTypes maybe.

Reply:
No it's not required here but is required in Match.
It is so because Match always shows column icon but Default doesn't.